src/aiortc/rtcrtptransceiver.py

```diff
@@ -54,7 +54,24 @@ def currentDirection(self) -> Optional[str]:
 
         One of `'sendrecv'`, `'sendonly'`, `'recvonly'`, `'inactive'` or `None`.
         """
-        return self._currentDirection
+        return self.__currentDirection
+
+    @currentDirection.setter
+    def currentDirection(self, direction: str) -> None:
```

Comment:
This looks wrong, `currentDirection` is supposed to be read-only:

https://developer.mozilla.org/en-US/docs/Web/API/RTCRtpTransceiver

Reply:
I would probably go for a `_setCurrentDirection` method to make it clear this is not public API

Reply:
I renamed the setter from `currentDirection` to `_currentDirection`